docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -0,0 +1,64 @@
+<button uxFocusIndicator
+        tabindex="0"
+        class="icon-demo btn btn-link button-secondary btn-text"
+        [uxPopover]="iconCodeSnippet"
+        popoverClass="icon-code-snippet-popover"
+        placement="bottom">
+    <i class="{{ iconClass }} p-r-xs" [ngClass]="classname"></i>
+    <span>{{ iconset === 'ux-icons' ? name : classname }}</span>
+</button>
+
+<ng-template #iconCodeSnippet>
+    <table class="icon-code-snippets-table">
+        <tbody>
+            <tr *ngIf="iconset === 'ux-icons'">
+                <td class="p-l-xs">
+                    <code>
+                        &lt;ux-icon name="{{name}}"&gt;&lt;/ux-icon&gt;
+                    </code>&nbsp;&nbsp;
+                </td>
+                <td>
+                    <i class="ux-icon ux-icon-clipboard m-r-xs"
+                       [uxTooltip]="tooltipTemplate"
+                       (click)="copy('ux-icon-component')">
+                    </i>
+                </td>
+            </tr>
+            <tr *ngIf="iconset === 'ux-icons'">
+                <td class="p-l-xs">
+                    <code>
```

Comment:
Would it be possible to try and use this style of code snippet? We normally use this style for actual code and the `<code>` element for keywords or classes etc.. rather than actual full lines of code

![image](https://user-images.githubusercontent.com/20795331/62453803-01bc1d80-b76b-11e9-97f6-c1441e44ad45.png)

Reply:
Changed style of code snippet